src/routing/bootstrap.rs

```diff
@@ -643,7 +649,10 @@ mod tests {
             task::yield_now().await;
```

Comment:
This `yield_now` (and all the others in this file) might not be necessary after the switch from `try_recv` to `recv`. A disadvantage of this switch is that this test can now hang if there is a bug where previously it would fail normally. Probably something we have to live with as `try_recv` has been removed from tokio, if I understand it correctly.

Reply:
I think we still need some of the occourences since we use `try_send()` in some places